content/en/docs/how-tos/adding-a-new-secret-to-ci.md

```diff
@@ -4,9 +4,66 @@ date: 2020-12-21T10:08:01-04:00
 
 Log into `api.ci` and create the secret in the namespace chosen above.
 
 Use the secret as the source in [the mapping file](https://github.com/openshift/release/tree/master/core-services/secret-mirroring).
 The secret and any modification on it afterwards will be populated to the targeting namespace on [all clusters](/docs/getting-started/useful-links/#clusters) in the build farm.
+
+```yaml
+- from:
+    namespace: some-namespace
+    name: source-some-secret
+  to:
+    namespace: test-credentials # the suggested namespace holding all secrets used in a step
+    name: target-some-secret # ensure the secret name unique cross all teams in our organization
+```
+
+## Use a secret in a job
+
+* The most common case is to use secrets in a [step](/docs/architecture/step-registry/#step). In this case, we suggest the user to mirror secrets to `test-credentials` namespace. The pod which runs the step can access the secrets defined in `credentials` stanza of the step definition. see [injecting-custom-credentials](https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-custom-credentials) for details.
+
+* For a job which is generated from `ci-operator`'s config and does not use steps, we have to use `ci` as the targeting namespace in secret-mirroring. And then mount the secrets via `secrets` stanza in the ci-operator's config, e.g.,
+
+```yaml
+tests:
+- as: "vet"                  # names this test "vet"
+  commands: "go vet ./..."   # declares which commands to run
+  container:
+    from: "src"              # runs the commands in "pipeline:src"
+  secrets:
+  - mount_path: /secret      # mount path of the extracted files from the secret
+    name: secret-name-in-ci  # the secret name in the ci namespace
+```
+
+* For a job which does not even use `ci-operator` at all, i.e. [handcrafted jobs](/docs/how-tos/contributing-openshift-release/#handcrafted-jobs), `ci` is the targeting namespace in secret-mirroring. As stated there, **creating handcrafted jobs is discouraged**. The follow example shows how to use secrets in a job definition:
+
+```yaml
+postsubmits:
+  org/repo:
+  - name: bar-job
+    branches:               # Regexps, only run against these branches.
+    - ^master$         
+    spec:                   # Valid Kubernetes PodSpec.
+      containers:
+      - image: docker.io/hello-world
+        name: ""
+        volumeMounts:
+        - mountPath: /secret             # mount path of the extracted files from the secret
+          name: volume-name
+      volumes:
+      - name: volume-name
+        secret:
+          secretName: secret-name-in-ci  # the secret name in the ci namespace
+```
+
+## FAQ
+
+### Why not use app.ci yet for maintaining the source secrets?
```

Comment:
This is kind of a leading question - I don't really think many users will have it.  Maybe something like ...

```
### I heard that the api.ci cluster is deprecated, why does secret mirroring still run there?

We are in the process of turning down the cluster, but cannot migrate the current secret mirroring process. A new solution for secret management will be in place before the cluster is fully removed.
```

Reply:
Removed FAQ.
Put it as an inline note intead.